### PR TITLE
fix(tutorials): Bare Metal Server with CLI MTA-1460

### PR DESCRIPTION
**Pull request by nedaehm**

### CODE REVIEW

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 1]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
 <Message type="requirement">
 
 - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com) 
-- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-- You have [generated your API key](/console/my-project/how-to/generate-api-key)
+- You have [configured your SSH Key](https://www.scaleway.com/en/docs/console/my-project/how-to/create-ssh-key/)
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:19]: same for the next one
```suggestion
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 2]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
...
 
 ## Configuring the CLI
 
-1. You must configure the Scaleway CLI on your machine. Open a terminal and type the following command:
+1. Open a terminal and type the following command to configure the Scaleway CLI on your machine:
 
-  ```
-  scw init
-  ```
+    `scw init`
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:29]:
```suggestion
    ```
    scw init
    ```
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 3]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
...
-  
-  The help info for the `zone` argument says that the default zone is `fr-par-2`. In fact, `fr-par-2` is the only zone in which Bare Metal servers are available at the moment, but it is not necessarily the default zone in your configuration. To create a Bare Metal server, you need to either:
-  a) change the default zone in your configuration file (check the location of this file in the output of sw init and change the value for `default zone` to `fr-par-2`), or
-  b) explicitly target the fr-par-2 zone in the commands to create and manage your server.
 
-  In this tutorial, we’ll target the zone explicitly in our commands.
+1. Type the following command in your terminal to create your Bare Metal Server:
 
-  </Message>
+  `scw baremetal server create name=name-of-your-server type=GP-BM1-L zone=fr-par-2`
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:94]:
```suggestion
  ```
  scw baremetal server create name=name-of-your-server type=GP-BM1-L zone=fr-par-2
  ```
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 4]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
...
+
+  </Message>
+
+2. Write down the ID of your Bare Metal server, as you will need it in the next steps.
 
   <Message type="note">
-  
-  You can view a summary of information for your Bare Metal server, and details for any other Bare Metal servers you have previously created, by typing `scw baremetal server list zone=fr-par-2`. For full information, use the command `scw baremetal server get <Server ID> zone=fr-par-2`
+
+    * Type `scw baremetal server list zone=fr-par-2` for a summary of information for your Bare Metal server, and details for other Bare Metal servers you have previously created.
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:125]: you can sometimes replace Bare Metal server by just server, so the full name repeats less.

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 5]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
...
-  You will see an output like:
 
-  ```
-  ID                 NAME                    CREATED AT  UPDATED AT  PROJECT ID
-  <YOUR SSH KEY ID>  <NAME OF YOUR SSH KEY>  1 week ago  1 week ago  <YOUR PROJECT ID>         
-  ```
+3. Type the following command:
 
-4. Note down your SSH Key ID.
+    `scw account ssh-key list`
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:163]:
```suggestion
    ```
    scw account ssh-key list
    ```
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 6]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
...
+
+
+    <Message type="tip">
+    
+    If you have multiple SSH keys, you can use the argument `all-ssh-keys=true` to install all your SSH keys on the server.
+
+    </Message>
+
+
+An output similar to the following displays:
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:186]:
```suggestion
  An output similar to the following displays: 
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 7]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
...
+Type the following command:
+
+```
+scw baremetal server reboot your-bare-metal-server-ID zone=fr-par-2
+```
+
+
+<Message type="note">
+
+  The `PingStatus`displays "down".
```
Comment (ldecarvalho-doc) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:294]:
```suggestion
  The `PingStatus` displays "down". 
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 8]
```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
 <Message type="requirement">
 
 - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com) 
-- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
-- You have [generated your API key](/console/my-project/how-to/generate-api-key)
+- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+- You have [generated your API key](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/)
```
Comment (nedaehm) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:20]:
```suggestion
- You have [generated your API key](/console/my-project/how-to/generate-api-key/)
```

File: compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx [hunk 9]
```diff
@@ -42,348 +16,354 @@ You can create and manage your Bare Metal servers from the console, via the API
...
 
--**To power on** your Bare Metal Server, use the following command:
+```
+scw baremetal server start your-bare-metal-server-ID zone=fr-par-2
+```
 
-  ```
-  scw baremetal server start <Bare Metal Server ID> zone=fr-par-2
-  ```
+<Message type="note">
```
Comment (RoRoJ) [inline compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx:259]: I think the last sections looks a bit message-box-heavy. What do you think about removing the message boxes around the bits that show the result of each command?

**Review by RoRoJ — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**